doc/source/user/index.rst

```diff
@@ -0,0 +1,23 @@
+:orphan:
+
+.. _user:
+
+################
+NumPy User Guide
+################
+
+This guide is intended as an introductory overview of NumPy and
+explains how to install and make use of the most important features of
+NumPy. For detailed reference documentation of the functions and
+classes contained in the package, see the :ref:`reference`.
+
+.. toctree::
+   :maxdepth: 1
+
+   setting-up
+   quickstart
+   basics
+   misc
+   numpy-for-matlab-users
```

Comment:
I'm wondering if we should include a link to `tutorials_index.rst` here to simulate the `For users` section of the main docs page

Reply:
Good suggestion - the `howtos_index` and `absolute_beginners` are the two other documents that are missing from the latex user guide as well. I will add all three.